### ISSUE TITLE
How to use yhe Karaoke model

### ISSUE THREAD

**Issue by mc261670164** [tsurumeso/vocal-remover#169]
Thanks for your great open source work!
I had seen the Karaoke model used in UVR, and I wonder how can use the Karaoke model in this project. Thanks very much!

**Comment by Goldp-js** [comment]
Do you mean using the UVR models in this project?

**Comment by mc261670164** [comment]
Yes, I see in UVR that Vocal-Remover module can be used with Karaoke model. So I wonder how to use that model in this project. Can you give me some guidance? Thank you!



At 2023-12-26 23:37:36, "Goldp-js" ***@***.***> wrote:

Do you mean using the UVR models in this project?

**Comment by Goldp-js** [comment]
I had already tried using the UVR models in this project, since they have different architectures but the same .pth extension. I tried to find a way to adapt them, but a problem arose: the dimensions of the VR models are not compatible with this project.

**Comment by Goldp-js** [comment]
The model for this project is compatible with UVR, but the UVR models are not compatible with this project.

**Comment by aufr33** [comment]
Hi. I was creating this model for the UVR (VR fork). Since it uses modified tsurumeso code, it will only work correctly in UVR.

**Comment by MinzChan** [comment]
> Hi. I was creating this model for the UVR (VR fork). Since it uses modified tsurumeso code, it will only work correctly in UVR.

!python inference.py --gpu 0 --pretrained_model models/HP2-4BAND-3090_4band_2.pth  --input './input/test.wav'
not use vocal-remover baseline.pt but ultimatevocalremovergui HP2-4BAND-3090_4band_2.pth
Your VR fork project doesn't seem to work properly
